playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -248,7 +248,7 @@
   # This var must be set with -e on invocation, as it is not a per-host inventory var
   # and is evaluated early. Values such as "20%" can also be used.
   serial: "{{ openshift_upgrade_nodes_serial | default(1) }}"
-  any_errors_fatal: true
+  max_fail_percentage: "{{ openshift_upgrade_nodes_max_fail_percentage | default(0) }}"
```

Comment:
@sdodson I noticed that you used the same variable for both controls_plane and nodes.  I think it could be useful if these were separate values.

I would like to be able to fail on any control plane error.  IT's much more important than the nodes.  I understand the argument about numbers.  Like, if i have 3 (or even 10) masters, one fail would fail it.  I still like the idea of separate values.

Thoughts?